x-pack/plugins/observability_ai_assistant/server/service/index.ts

```diff
@@ -56,9 +57,7 @@ export class ObservabilityAIAssistantService implements IObservabilityAIAssistan
 
       const esClient = coreStart.elasticsearch.client.asInternalUser;
 
-      const fnv1a = await import('@sindresorhus/fnv1a');
```

Comment:
This wasn't working. Some stuff with ESM blablabla. Picked another library, doesn't make much of a difference.